scripts/bootstrap/termux-bootstrap-second-stage.sh

```diff
@@ -5,6 +5,9 @@ export TERMUX_PREFIX="@TERMUX_PREFIX@"
 export TERMUX_PACKAGE_MANAGER="@TERMUX_PACKAGE_MANAGER@"
 export TERMUX_PACKAGE_ARCH="@TERMUX_PACKAGE_ARCH@"
 
+TERMUX__USER_ID___N="@TERMUX_ENV__S_TERMUX@USER_ID"
+printf -v "TERMUX__USER_ID" "%s" "${!TERMUX__USER_ID___N}" || return $?
```

Comment:
1.) The indirection could use a comment.
2.) Couldn't you do this instead?
```suggestion
TERMUX__USER_ID="${!TERMUX__USER_ID___N}" || return $?
```

Reply:
Yeah, you are right, in other places I used that format, the `-v` variable is the `___N` one instead and I use a function for this case.

https://github.com/termux/termux-exec/blob/fc40c5aa464c70f523431fa0984a1c77420e9e92/tests/termux-exec-tests.in#L36